internal/pkg/cli/flag.go

```diff
@@ -213,6 +214,7 @@ Must be of the format '<keyName>:<dataType>'.`
 	taskRoleFlagDescription      = "Optional. The ARN of the role for the task to use."
 	executionRoleFlagDescription = "Optional. The ARN of the role that grants the container agent permission to make AWS API calls."
 	envVarsFlagDescription       = "Optional. Environment variables specified by key=value separated with commas."
+	secretsFlagDescription       = "Optional. Secrets to inject into the container. Specified by key=value separated with commas."
```

Comment:
nit: can we change "separated with commas" to "separated by commas" here and for the envVars flag?